src/components/timeline/TrackScreenshots.js

```diff
@@ -87,12 +120,25 @@ class Screenshots extends PureComponent<Props, State> {
       trackHeight,
     } = this.props;
     const { pageX, offsetX, containerTop } = this.state;
+    let payload: ScreenshotPayload | null;
+    if (offsetX !== null) {
+      const screenshotIndex = this.findScreenshotAtMouse(offsetX);
+      if (screenshotIndex === null) {
+        payload = null;
+      } else {
+        payload = (screenshots[screenshotIndex].data: any);
+      }
+    } else {
+      payload = null;
+    }
```

Comment:
Maybe an easier to read suggestion, by initializing payload at null at first:
```js
    let payload: ScreenshotPayload | null = null;
    if (offsetX !== null) {
      const screenshotIndex = this.findScreenshotAtMouse(offsetX);
      if (screenshotIndex !== null) {
        payload = (screenshots[screenshotIndex].data: any);
      }
    }
```
This way you avoid all the `else` blocks that aren't really useful, and the code is leaner and easier to follow (in my opinion). Tell me what you think!

Reply:
> This way you avoid all the else blocks that aren't really useful, and the code is leaner and easier to follow (in my opinion). Tell me what you think!

Thanks for pointing that out. You are right it makes the code more readable. I will definitely change that.